test/common.go

```diff
@@ -0,0 +1,131 @@
+package test
+
+import (
+	"bytes"
+	"embed"
+	cmpact "encoding/json"
+	"github.com/goccy/go-json"
+	"fmt"
+	manifestsdk "github.com/TBD54566975/ssi-sdk/credential/manifest"
+	"github.com/oliveagle/jsonpath"
+	"github.com/pkg/errors"
+	credmodel "github.com/tbd54566975/ssi-service/internal/credential"
+	"io"
+	"net/http"
```

Comment:
might have to run [goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports) here

Reply:
would suggest adding `goimports` to the golang-ci file :)